CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.11.0 Release
+- Release Date: August 16, 2018
+- Server Versions Supported: Server v4.0+ is required, Self-Signed SSL Certificates are not supported
+
+### Highlights
+
+#### Searching Archived Channels
+- Added ability to search for archived channels on desktop and mobile clients.
+
+#### Deep Linking
+- 
```

Comment:
This needs docs, they should be in the PR but we should include them in the mobile docs, this also needs a custom build.

Reply:
https://github.com/mattermost/docs/pull/2218